packages/avatar-group/test/visual/lumo/avatar-group.test.js

```diff
@@ -45,4 +45,11 @@ describe('avatar-group', () => {
     await nextRender(element);
     await visualDiff(document.body, 'opened');
   });
+
+  it('avatar-size', async () => {
+    element.items = [{ name: 'Abc Def' }, { name: 'Ghi Jkl' }, { name: 'Mno Pqr' }, { name: 'Stu Vwx' }];
+    element.style.cssText += '--vaadin-avatar-size: 45px';
```

Comment:
It's better to use `style.setProperty()` for custom CSS properties:

```suggestion
    element.style.setProperty('--vaadin-avatar-size', '45px');
```

Reply:
I actually tried that first, but it did not work. I got an exception that says the property cannot be found.